dotnet3.5/S3/ListObjectsExample/ListObjects/ListObjects.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.ListObjects]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace ListObjects
+{
+    // The following example will list the objects in an Amazon Simple Storage
+    // Service (Amazon S3) bucket. It was created using AWS SDK for .NET 3.5
+    // and .NET 5.0.
+    public class ListObjects
+    {
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the new bucket
```

Comment:
Comments for constants here seem unnecessary -- it seems obvious to me that users would need to change this to their own Region and bucket name, or else they will get unexpected results.

Reply:
Removed